### PR TITLE
update repo layout for new operator-sdk.

### PR DESCRIPTION
**Pull request by morvencao**
ref: https://github.com/open-cluster-management/backlog/issues/9608
also fix the base image issue - https://github.com/open-cluster-management/backlog/issues/11324

### CODE REVIEW

File: Dockerfile.prow [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Copyright (c) 2021 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project.
+FROM registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux-amd64 AS builder
```
Comment (clyang82) [inline Dockerfile.prow:3]: Do not use this image. In order to support s390x, the image must support multi-arch. so replace it with `xxx/builder:go1.15-linux`

**Review by clyang82 — CHANGES_REQUESTED** [review]

**Comment by clyang82** [comment]
Please remove `certmanager` `scorecard` etc. so far, we do not have plan to use them. We can regenerate if needed.

File: Dockerfile.prow (hunk 1 above)
Comment (morvencao) [reply Dockerfile.prow]: done

**Comment by morvencao** [comment]
removed all unused manifests under `config` directory @clyang82

**Pull request merged**